### ISSUE TITLE
globally set SourceMemberNamingConvention 

### ISSUE THREAD

**Issue by kotzer3** [MattHoneycutt/HeroicFramework#21]
Great tool, thanks Matt!

Ho to set globally the source naming convention to:
```      configuration.SourceMemberNamingConvention = new LowerUnderscoreNamingConvention();```

I don't want to insert it in every IMapFrom implementation. Currently I have:
```  public partial class client : EntityObjectBase, IMapTo<Client> { }
  public partial class configuration : EntityObjectBase, IMapTo<Configuration> { }
  public partial class permission : EntityObjectBase, IMapTo<Permission> { }
  public partial class role : EntityObjectBase, IMapTo<Role> { }
  public partial class user : EntityObjectBase, IMapFrom<User>, IHaveCustomMappings {
    public void CreateMappings(IMapperConfigurationExpression configuration)
    {
      configuration.SourceMemberNamingConvention = new LowerUnderscoreNamingConvention();
      configuration.CreateMap<user, User>()
        .ForMember(d => d.Roles, opt => opt.MapFrom(src => src.role))
        .ForMember(d => d.Clients, opt => opt.MapFrom(src => src.client));
    }
  }
```

**Comment by kotzer3** [comment]
I found out, that i used IMapFrom instead of IMapTo for user. This is fine now. But after injecting source of HeroicAutoMapper into my solution to better understand what exctly is going on, i found out, that the problem is my naming convention.

in databasebase I have `LowerUnderscoreNamingConvention` and in Domain I have `PascalCaseNamingConvention`.

To archive this (without HeroicAutoMapper) I used two profiles. one from eo (entityobjects) to do (domainobjects) with namingconventions:
```        
cfg.SourceMemberNamingConvention = new LowerUnderscoreNamingConvention();
cfg.DestinationMemberNamingConvention = new PascalCaseNamingConvention();
```

and from do to eo I used following naming conventions:
```        
cfg.SourceMemberNamingConvention = new PascalCaseNamingConvention();
cfg.DestinationMemberNamingConvention = new LowerUnderscoreNamingConvention();
```

Is there any way to stay using interfaces from HeroicAutomapper instead of using manual mappings again?
